### ISSUE TITLE
[Bug]: lakectl uri validations errors are not clear enough

### ISSUE THREAD

**Issue by N-o-Z** [treeverse/lakeFS#6815]
### What happened?

Reported by a community member:

```
lakectl merge --verbose lakefs://eu-fr/development/ lakefs://eu-fr/testintegrationnetwork/
Invalid source ref: not a valid ref uri
Error executing command.
```

It was not clear to the user what was invalid in the ref uri

### Expected behavior

Message should be clear, to allow the user to resolve invalid ref issue on their own

### lakeFS version

0.113.0

**Comment by TinoSM** [comment]
As the user:
Those URLs came from the UI "copy link" (later on I've found that I can merge using the UI but well...). 
<img width="1398" alt="image" src="https://github.com/treeverse/lakeFS/assets/821046/61a22b5d-67cd-47c8-96ba-a83eada838b2">


 The error itself is quite stupid, but I tried with double-quotes, was thinking if the issue was because of the "-" in the repository name, didn't think on "/" being the issue.

Given that merge only supports branches, maybe it can even just ignore the last "/" if it's there (or hint that the / at the end should be removed).
